examples/pylab_examples/load_converter.py

```diff
@@ -1,20 +1,30 @@
 from __future__ import print_function
-from matplotlib.dates import bytespdate2num
-#from matplotlib.mlab import load
 import numpy as np
-from pylab import figure, show
+import matplotlib.pyplot as plt
 import matplotlib.cbook as cbook
+import matplotlib.dates as mdates
+
+
+# Note: matplotlib.dates doesn't have bytespdate2num.
```

Comment:
Very weird, I cannot make the import statements work, though. I get the error: 

    In [9]: import matplotlib.dates as mdate

    In [10]: mdate.bytespdate2num()
    ---------------------------------------------------------------------------
    AttributeError                            Traceback (most recent call last)
    <ipython-input-10-7c2709974f11> in <module>()
    ----> 1 mdate.bytespdate2num()

    AttributeError: 'module' object has no attribute 'bytespdate2num'

Any idea what's going on? Is the import statement not done correctly?